hibikase/MainWindow.cpp

```diff
@@ -38,6 +38,22 @@ MainWindow::MainWindow(QWidget* parent) :
 {
     ui->setupUi(this);
 
+    connect(ui->menuEdit, &QMenu::aboutToShow, [this]() {
+        QMenu *coolMenu = ui->mainLyrics->MakeMenu();
```

Comment:
```suggestion
        QMenu* menu = ui->mainLyrics->MakeMenu();
```

(or some other name, like `editMenu`)